### ISSUE TITLE
How to check if the task was successful?

### ISSUE THREAD

**Issue by jclsn** [skywind3000/asynctasks.vim#105]
I would like to run a command after a task was run successfully. Is there a way?

**Comment by skywind3000** [comment]
https://github.com/skywind3000/asynctasks.vim/wiki/Task-Examples#silence-and-notify

**Comment by jclsn** [comment]
Yeah that works, but how can I now start the debugger afterwards? I would like to use the task as a pre-launch task for Vimspector. I have tried to use ``notify=:call vimspector#Launch()`` but it doesn't work.

**Comment by jclsn** [comment]
Any solution to this? I would like to add Async Tasks to the pre-launch strategies for Vimspector as an alternative to dispatch.vim, but for that I need the return value.

**Comment by skywind3000** [comment]
what is your task configuration ?

**Comment by jclsn** [comment]
I just realized that I was still using dispatch.vim's ``:Make`` as building command. Anyway, it should not matter. I just need a way to determine if a command was successful. If asynctasks.vim has no influence on what the command returns, which I am assuming, this is pointless to try I guess.

Anyway, this is what I was trying

```
[.build-and-debug]
command=Make
silent=1
notify=:call vimspector#Launch()
```

**Comment by skywind3000** [comment]
你要检查下，powershell 里 `$function:prompt` 初始化上了没有？见：
https://github.com/skywind3000/z.lua/blob/0992ebf9f1f6cdaa114e65d3aa76bfb1bd9a6fd4/z.lua#L2662

会不会被其他不讲武德的插件给覆盖了？
